### PR TITLE
added the usage of std::find

### PR DESCRIPTION
**Pull request by glanderson42**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/cplusplus/vector/index.md [hunk 1]
```diff
@@ -233,3 +233,50 @@ int main(){
 ```
 
 You can also sort in descending using lamda like the one above.
+
+### Finding an element in a ```std::vector```
+Before C++11:
+```c++
+#include <iostream>
```
Comment (scissorsneedfoodtoo) [inline guide/english/cplusplus/vector/index.md]: Running this code produces the following error: `error: no matching function for call to 'find'`

File: guide/english/cplusplus/vector/index.md [hunk 2]
```diff
@@ -233,3 +233,50 @@ int main(){
...
+    else
+    {
+        std::cout << "Element not found :( \n";
+    }
+}
+```
+
+After C++11:
+```c++
+#include <iostream>
```
Comment (scissorsneedfoodtoo) [inline guide/english/cplusplus/vector/index.md]: This code produces the same error mentioned above.

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]
I got some errors while checking your code. Could you fix those and update this PR?

**Comment by RandellDawson** [comment]
@scissorsneedfoodtoo This PR has gone stale.

**Comment by scissorsneedfoodtoo** [comment]
@RandellDawson, I agree. Best to close this.

**Pull request closed without merging**